Workbooks/Virtual Machines/Virtual Machine Performance/Single Virtual Machine Performance.workbook

```diff
@@ -0,0 +1,515 @@
+{
+  "version": "Notebook/1.0",
+  "isLocked": false,
+  "items": [
+    {
+      "type": 1,
+      "content": {
+        "json": "# Single Virtual Machine Performance\n\n*Click on a row to see trends for only that machine*\n---"
+      },
+      "conditionalVisibility": null
+    },
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "query": "",
+        "crossComponentResources": [
+          "{Workspace}"
+        ],
+        "parameters": [
+          {
+            "id": "8f72709f-725a-41e6-82a2-8d067e38a261",
+            "version": "KqlParameterItem/1.0",
+            "name": "Workspace",
+            "type": 5,
+            "isRequired": true,
+            "isHiddenWhenLocked": false,
+            "typeSettings": {
+              "resourceTypeFilter": {
+                "microsoft.operationalinsights/workspaces": true
+              },
+              "additionalResourceOptions": []
+            },
+            "value": "/subscriptions/f9cf50ed-d814-4b95-b76c-c709845f84c9/resourceGroups/mms-eus/providers/Microsoft.OperationalInsights/workspaces/admdemo"
```

Comment:
in a template, you'll want to remove specific values from parameters.  for resource params, you might want to add "any one" options so that by default this will inherit whatever values are available.   (this template would show a value but not work for people who can't see admdemo!)

Reply:
as a test, sign into some other random azure account (like a personal one that doesn't have access to admdemo, and paste your template into advanced mode and see if it works.  that's how I test most people's templates.  @ericc1103 we need a way in the portal to point workbooks at a pull request somehow and have it show whatever templates are in that PR?

Reply:
@gardnerjr Good suggestion. Could you add it into the backlog since you have the list?

Reply:
Thank you for the comments and detailed instructions on how to test, I have tested it on a separate account with a different subscription.